spec/config.yaml

```diff
@@ -639,6 +649,196 @@ mappings:
       perm: {type: keyword}
       permusers: {type: keyword}
 
+  "sample_set_1":
+    global_mappings: [ws_auth, ws_object]
+    properties:
+      description: {type: text}
+      sample_ids: {type: keyword}
+      sample_names: {type: keyword}
+      sample_versions: {type: integer}
+
+  "sample_set_version_1":
+    global_mappings: [ws_auth, ws_object]
+    properties:
+      description: {type: text}
+      sample_ids: {type: keyword}
+      sample_names: {type: keyword}
+      sample_versions: {type: integer}
+
+  "sample_1":
+    global_mappings: [ws_auth, ws_object]
+    properties:
+      all_sample_metadata_field: {type: text}
+      sample_set_ids: {type: keyword}
+      save_date: {type: text, copy_to: all_sample_metadata_field}
```

Comment:
Do you really need this `all_sample_metadata_field`? I feel like you could just use a wildcard in the query:

`{"match": {"*": "my text here"}}`

Reply:
We talked about it on slack. Seems fine. Might want to prefix the additional user fields though with something like "user_x"